lib/web_apis/api_importer.es6.js

```diff
@@ -16,31 +16,72 @@
  */
 'use strict';
 
+require('./browser.es6');
+require('./web_interface.es6');
+require('./version_history.es6');
+
 foam.CLASS({
   name: 'ApiImporter',
   package: 'org.chromium.apis.web',
   documentation: `API Importer is a class that handles importing browserAPIs
     from webCatalog object to browserAPI DAO.`,
   requires: [
-    'org.chromium.apis.web.BrowserAPI',
+    'org.chromium.apis.web.Browser',
+    'org.chromium.apis.web.WebInterface',
+    'org.chromium.apis.web.BrowserWebInterfaceJunction',
+    'org.chromium.apis.web.VersionHistory',
     'foam.mlang.ExpressionsSingleton',
     'foam.dao.EasyDAO',
   ],
+  exports: [
+    'browserDAO',
+    'interfaceDAO',
+    'browserApiDAO as browserWebInterfaceJunctionDAO',
```

Comment:
The reason to do this is when browser.interface.put() is called, it will look for a DAO named 'browserWebInterfaceJunctionDAO'.